builder/llpcBuilder.cpp

```diff
@@ -90,6 +90,19 @@ Builder::~Builder()
     delete m_pPipelineState;
 }
 
+// =====================================================================================================================
+// Get the type pElementTy, turned into a vector of the same width as pMaybeVecTy if that is a vector type
+Type* Builder::getScalarOrVectorTy(
```

Comment:
I think we need a better name for this fucntion, maybe Rex have more comment about it.

Reply:
ConditionallyVectorizeTy?

Reply:
Actually it would be more consistent to have a "get". So GetConditionallyVectorizedTy?

Reply:
I am OK with this name GetConditionallyVectorizedTy. The original name is somewhat unclear to me until I see it is used for Quantize.